src/java.desktop/share/classes/javax/imageio/metadata/IIOMetadata.java

```diff
@@ -398,12 +396,9 @@ public IIOMetadataFormat getMetadataFormat(String formatName) {
             throw new IllegalArgumentException("Unsupported format name");
         }
         try {
-            final String className = formatClassName;
             // Try to load from the module of the IIOMetadata implementation
             // for this plugin since the IIOMetadataImpl is part of the plugin
-            PrivilegedAction<Class<?>> pa = () -> { return getMetadataFormatClass(className); };
-            @SuppressWarnings("removal")
-            Class<?> cls = AccessController.doPrivileged(pa);
+            Class<?> cls =  getMetadataFormatClass(formatClassName);
```

Comment:
```suggestion
            Class<?> cls = getMetadataFormatClass(formatClassName);
```

Reply:
ok. But do you have any comments on the *functional* aspect of this PR ?